os_net_config/impl_nmstate.py

```diff
@@ -50,6 +50,27 @@
 # Import the raw NetConfig object so we can call its methods
 netconfig = os_net_config.NetConfig()
 
+_VF_BIND_DRV_SCRIPT = '''
+dpdk_vfs="{dpdk_vfs}"
+linux_vfs="{linux_vfs}"
+set -x
```

Comment:
To ensure we don't have any surprises, also disable `errexit`:

```bash
set +e
```

Reply:
done